src/libm-tester/CMakeLists.txt

```diff
@@ -1,14 +1,17 @@
 # Note: We are assuming SLEEF is the CMake root project.
 # TODO: Remove constraint: do not use CMAKE_BINARY_DIR and CMAKE_SOURCE_DIR
 link_directories(${CMAKE_BINARY_DIR}/lib)                 # libsleef
+link_directories(${CMAKE_BINARY_DIR}/src/common)          # common.a
 include_directories(${CMAKE_BINARY_DIR}/include)          # sleef.h
 include_directories(${CMAKE_SOURCE_DIR}/src/libm)         # rename.h
 include_directories(${CMAKE_BINARY_DIR}/src/libm/include) # rename headers
 
 set(LIB_SLEEF sleef)
 find_library(LIB_MPFR mpfr)
 set(EXTRA_LIBS m
-  ${LIB_MPFR})
+  ${LIB_MPFR}
+  common
```

Comment:
Sorry I missed this one. Please rename the `common` target with a variable, say `${LIB_COMMON_STATIC}`.